account_chart_update/wizard/wizard_chart_update.py

```diff
@@ -0,0 +1,1320 @@
+            dest_id = self.find_account_by_templates(tpl.account_dest_id)
+            existing = self.env["account.fiscal.position.account"].search(
+                [
+                    ("position_id", "=", pos_id),
+                    ("account_src_id", "=", src_id),
+                    ("account_dest_id", "=", dest_id),
+                ]
+            )
+            if not existing:
+                # create a new mapping
+                result.append(
+                    (
+                        0,
+                        0,
+                        {
+                            "position_id": pos_id,
+                            "account_src_id": src_id,
+                            "account_dest_id": dest_id,
+                        },
+                    )
+                )
+            else:
+                current_fp_accounts -= existing
+        # Mark to be removed the lines not found
+        if current_fp_accounts:
+            result += [(2, x.id) for x in current_fp_accounts]
+        return result
+
+    @tools.ormcache("templates", "current_fp_taxes")
+    def find_fp_tax_by_templates(self, templates, current_fp_taxes):
+        result = []
+        for tpl in templates:
+            pos_id = self.find_fp_by_templates(tpl.position_id)
+            src_id = self.find_tax_by_templates(tpl.tax_src_id)
+            dest_id = self.find_tax_by_templates(tpl.tax_dest_id)
+            existing = self.env["account.fiscal.position.tax"].search(
+                [
+                    ("position_id", "=", pos_id),
+                    ("tax_src_id", "=", src_id),
+                    ("tax_dest_id", "=", dest_id),
+                ]
+            )
+            if not existing:
+                # create a new mapping
+                result.append(
+                    (
+                        0,
+                        0,
+                        {
+                            "position_id": pos_id,
+                            "tax_src_id": src_id,
+                            "tax_dest_id": dest_id,
+                        },
+                    )
+                )
+            else:
+                current_fp_taxes -= existing
+        # Mark to be removed the lines not found
+        if current_fp_taxes:
+            result += [(2, x.id) for x in current_fp_taxes]
+        return result
+
+    @api.model
+    @tools.ormcache("name")
+    def fields_to_ignore(self, name):
+        """Get fields that will not be used when checking differences.
+
+        :param str template: A template record.
+        :param str name: The name of the template model.
+        :return set: Fields to ignore in diff.
+        """
+        specials_mapping = {
+            "account.tax.template": {"chart_template_id", "children_tax_ids"},
+            "account.account.template": {"chart_template_id", "root_id", "nocreate"},
+            "account.fiscal.position.template": {"chart_template_id"},
+        }
+        specials = {
```

Comment:
```suggestion
        # Exclude `mail.thread` fields
        specials = set(self.env['mail.thread']._fields) | {
```

I've seen the problem is when trying to replace data on the `mail.message` fields.

I suggest to add all fields of the `mail.thread` model onto the fields to ignore. 
In previous versions, `account.account.template` doesn't inherit from `mail.thread`, and these fields makes not sense to override existing data. Maybe someone put some important messages on this model and we don't want to overwrite them.

Reply:
Done